src/main/java/com/elixr/poc/service/UserService.java

```diff
@@ -0,0 +1,33 @@
+package com.elixr.poc.service;
+
+import com.elixr.poc.repository.UserRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+/*
+Service layer
+ */
+@Service
+public class UserService {
+
+    private final UserRepository userRepository;
+
+    public UserService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    /*
+    Deleting the user by the userId.
+     */
+    public boolean deleteUserDetails(UUID userId) {
+        boolean success = false;
+        try {
+            userRepository.deleteById(userId);
```

Comment:
Its better to check if entity exists by calling repository.findById(uuid) first.
If not found then display an error message  "No matching entity found"

Reply:
Updated.